setup.py

```diff
@@ -19,12 +30,10 @@
         "License :: OSI Approved :: MIT License",
         "Operating System :: OS Independent",
     ],
-    install_requires=[
-        "fastavro",
-        "more-itertools",
-        "numpy",
-        "pandas",
-        "requests",
-        "six",
-    ],
+    install_requires=requires,
+    python_requires=">=3.5",
```

Comment:
unrelated to the docs story, but I saw this in the requests library setup.py and it should help us enforce the python version of our clients